tests/components/rfxtrx/test_connect.py

```diff
@@ -0,0 +1,39 @@
+"""Test that we correctly connect using the pyRFXtrx library."""
```

Comment:
We generally try to match the test_x name with the main file it tests. So i think it make more sense to put this in test_init.py

Reply:
Gotta be clean :)

Done.